docs/cloud/visualize/overview.md

```diff
@@ -1,10 +1,12 @@
 # Home, overview and single node views
 
+The Home view has details for the War Room, the Overview uses charts from all the nodes in the War Room and the Single Node view has charts for a specific Node.
```

Comment:
```suggestion
Learn how to use the Home tab to get high level information for each War Room, the Overview tab to explore your infrastructure with charts from all the nodes in the War Room and the Single Node view to focus on a specific Node.
```

Reply:
@cakrit this is longer than 160 characters, ahrefs complains for everything more than 160

Reply:
```suggestion
Learn how to use the Home, Overview, and Single Node tabs in Netdata Cloud, to explore your infrastructure and troubleshoot issues.
```